action.yml

```diff
@@ -40,9 +40,17 @@ inputs:
     description: 'create or destroy'
     required: false
     default: 'create'
+  USE_PULUMI:
```

Comment:
maybe make this a string instead of a bool -- and they can pick the IaC tool?

Reply:
Great suggestion. I updated it to be IAC_TYPE, and listed the current possible choices in the description